### DISCUSSION TITLE
Max number of simultaneous connections?

### DISCUSSION THREAD

**Discussion by Wakeboardsam** [RaspAP/raspap-webgui#839] [Q&A]
Is there a max number of simultaneous connections I can use for raspap?
I'm using a rpi4 b / + what ever the current model is.
Got the client ap working, pass through internet for the connections working,all using the SOC wifi. 
Weird thing is, I've got 5 devices, only 3 of them will connect at any one time, disconnect one (toggle wifi /forget wifi), one of the other devices will connect. 
It's an odd thing and am wondering if it's just me or if this is normal behavior, or if there is a way to increase # of connections.
I'm a noob, and like to learn but much of the cli world is foreign to me. Which is why I was excited about raspap gui, thank you!
Sam.

**Comment by billz** [comment]
Short answer: it depends.
Longer answer: discussed extensively in https://github.com/RaspAP/raspap-webgui/issues/462 and https://github.com/raspberrypi/linux/issues/3010

**Reply by Wakeboardsam** [reply]
Thank you!
I obviously searched the wrong terms before posting.
Genuinely appreciate your time to reply.

**Reply by billz** [reply]
I felt this could be better summarized with a new FAQ https://docs.raspap.com/faq/#maxclients
Thanks for raising this question.
